state/txindex/kv/kv_test.go

```diff
@@ -52,7 +52,7 @@ func benchmarkTxIndex(txsCount int, b *testing.B) {
 	b.ResetTimer()
 
 	for n := 0; n < b.N; n++ {
-		err = indexer.AddBatch(batch)
```

Comment:
this may be important for the benchmark (will compiler eliminate the function call?)

if not, let's remove `_ =` altogether